Moonshot/room_templates/room_scenes/RouteRooms/route_LD_james14.tscn

```diff
@@ -0,0 +1,105 @@
+[gd_scene load_steps=10 format=2]
```

Comment:
Should be DL not LD )

Reply:
https://github.com/kepler471/moonshot/pull/258/commits/6c9855f518a6830fccf5607f8c3688afdb84ca54